modules/EmailTemplates/EmailTemplate.php

```diff
@@ -750,7 +750,13 @@ protected static function add_replacement($data, $field_def, $replacement)
         foreach ($replacement as $key => $value) {
             // @see defect #48641
             if ('multienum' == $field_def['type']) {
-                $value = implode(', ', unencodeMultienum($value));
+                // Issue #2802
```

Comment:
Please remove this comment. Comments should explain the "why" of a code block. The issue number should be in the git commit. So that when a developer runs git blame they can see the reference there.